<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Password</class>
 <widget class="QDialog" name="Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>218</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sicherheitsfrage 1: Wie lautet </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sicherheitsfrage 2: Wie lautet </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEUsername">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lES1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lES2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCheck">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(0, 121, 0);
}
</string>
   </property>
   <property name="text">
    <string>Überprüfe Daten</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E-Mail:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCheck_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);
}
</string>
   </property>
   <property name="text">
    <string>Username vergessen?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>670</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(255, 170, 0);
}
</string>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEPasswort">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neues Passwort:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>821</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="berschhirft">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Du hast dein Passwort vergessen? Gar kein Problem!&lt;br/&gt;Gib einfach deine Daten ein und dann kannst du ein neues Passwort festlegen.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>der Geburtsname deiner Mutter?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>dein Geburtsjahr?</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lEUsername</zorder>
  <zorder>lES1</zorder>
  <zorder>lES2</zorder>
  <zorder>ButtonCheck</zorder>
  <zorder>label_4</zorder>
  <zorder>Email</zorder>
  <zorder>ButtonCheck_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>lEPasswort</zorder>
  <zorder>label_5</zorder>
  <zorder>lbl3</zorder>
  <zorder>lbl2</zorder>
  <zorder>lbl4</zorder>
  <zorder>berschhirft</zorder>
  <zorder>lbl1</zorder>
  <zorder>label_12</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
